### PR TITLE
D modes for Team900

### PR DESCRIPTION
**Pull request by JyeSmith**
Adds D50 for 900/433Mhz.  This uses 200Hz OTA modulation parameters and 4 repeat packets over 3 frequencies (similar to D250 for 2.4GHz).  This doubles when used in conjunction with Gemini.

Done in a janky way for compatibility with V3, but will ideally be ordered correctly as part of V4.

Do we want to consider any other modes?

### CODE REVIEW

**Review by wvarty — APPROVED** [review]
Looks fine.
I hate the fact that packet rates with higher hz are on the "wrong" end of the menu though :|

File: src/src/common.cpp [hunk 1]
```diff
@@ -11,14 +11,16 @@ expresslrs_mod_settings_s ExpressLRS_AirRateConfig[RATE_MAX] = {
...
+    {5, RADIO_TYPE_SX127x_LORA, RATE_DVDA_50HZ,      SX127x_BW_500_00_KHZ, SX127x_SF_6, SX127x_CR_4_7, TLM_RATIO_1_64, 2,  5000,  8, OTA4_PACKET_SIZE, 4}};
 
 expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
     {0, RATE_LORA_200HZ,     -112,  4380, 3000, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(3.0)},
     {1, RATE_LORA_100HZ_8CH, -112,  6690, 3500, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(3.0)},
     {2, RATE_LORA_100HZ,     -117,  8770, 3500, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(2.5)},
     {3, RATE_LORA_50HZ,      -120, 18560, 4000, 2500, 600, 5000, SNR_SCALE(-1), SNR_SCALE(1.5)},
-    {4, RATE_LORA_25HZ,      -123, 29950, 6000, 4000,   0, 5000, SNR_SCALE(-3), SNR_SCALE(0.5)}};
+    {4, RATE_LORA_25HZ,      -123, 29950, 6000, 4000,   0, 5000, SNR_SCALE(-3), SNR_SCALE(0.5)},
+    {0, RATE_DVDA_50HZ,      -112,  4380, 3000, 2500, 600, 5000, SNR_SCALE( 1), SNR_SCALE(3.0)}};
```
Comment (pkendall64) [inline src/src/common.cpp:23]: I know the index isn't used but if we are having it in the structure then it should at least be correct. So it should be 5 at the start 😄

Comment (JyeSmith) [reply src/src/common.cpp]: Suppose 😄

**Review by pkendall64 — APPROVED** [review]

**Comment by paveltar** [comment]
Does it use LoRa or FLRC (even possible for Team900?) ?

**Comment by JyeSmith** [comment]
Lora, and the same RF parameters as 200Hz.

FLRC isn't available on 900.

**Pull request merged**